modules/json-schema/src/internals/JsonSchemaToIModel.scala

```diff
@@ -80,9 +80,9 @@ private class JsonSchemaToIModel[F[_]: Parallel: TellShape: TellError](
     val schemaName = Name(schemaNameSegment)
 
     // Computing schemas under the $defs field, if it exists.
-    def $defSchemas = {
+    def $defSchemas: String => Vector[Local] = { name =>
```

Comment:
since this is a `def` already, is there a reason to return a function as opposed to using `def $defSchemas(name: String)` syntax?